### PR TITLE
coprocessor: support expression.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: coprocessor.go [hunk 1]
```diff
@@ -22,23 +22,36 @@ import (
...
-	//	case kv.ReqTypeSelect:
-	//		return subType == kv.ReqSubTypeBasic
-	//	case kv.ReqTypeIndex:
-	//		return subType == kv.ReqSubTypeBasic
-	//	}
+	switch reqType {
+	case kv.ReqTypeSelect:
+		return supportExpr(tipb.ExprType(subType))
+	case kv.ReqTypeIndex:
+		return subType == kv.ReqSubTypeBasic
```
Comment (qiuyesuifeng) [inline coprocessor.go]: Should handle not supported type as error return?

Comment (coocood) [inline coprocessor.go]: Not support return false is enough.

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**